### DISCUSSION TITLE
Welcome to custom-cdj Discussions!

### DISCUSSION THREAD

**Discussion by LorenzoBalducci96** [LorenzoBalducci96/custom-cdj#6] [General]
## 👋 Welcome!
  We’re using Discussions as a place to discuss problems and ideas related to the project.
Here you can ask opinions for realizing your own setup, you can ask help if you have problems making you own custom cdj or if you have some perplexity with the official documentation.

Here we can also discuss some related projects (all in one custom midi controller, or something similar).

**Comment by Jako420** [comment]
Can I also build this with a Discovery kit (STM32F746NG)?
